webapp/src/app.tsx

```diff
@@ -921,41 +841,47 @@ export class ProjectView
         );
     }
 
-    openHome() {
-        this.stopSimulator();
-        // clear the hash
-        pxt.BrowserUtils.changeHash("", true);
-        this.setState({ home: true });
-        this.allEditors.forEach(e => e.setVisible(false));
-        this.homeLoaded();
-    }
-
-    private homeLoaded() {
-        pxt.tickEvent('app.home');
+    importFile(file: File) {
```

Comment:
Maybe this should move too to importhelpers

Reply:
It relies on resourceImporters which is attached to the editor (for extensions), so we'd have to pass it around or just keep the method there.